app/helpers/decidim/consultations/consultations_helper.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Consultations
+    module ConsultationsHelper
```

Comment:
This helper already exists in decidim decidim, You can make a decorator by adding only the necessary functionality

Reply:
Thanks. Done.